vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -76,7 +76,15 @@
       </Term>
       <Term Name="ApplySupportedDefaults" Type="Aggregation.ApplySupportedBase" Nullable="false" AppliesTo="EntityContainer">
         <Annotation Term="Core.Description">
-          <String>Default support of the `$apply` system query option for all collection-valued resources in the container. Annotating term `ApplySupported` for a specific collection-valued resource overrides the default setting</String>
+          <String>Default support of the `$apply` system query option for all collection-valued resources in the container</String>
+        </Annotation>
+        <Annotation Term="Core.LongDescription">
+          <String>Annotating term [`ApplySupported`](#ApplySupported) for a specific collection-valued resource overrides the default setting with the specified properties using PATCH semantics:
+
+- Properties specified in `ApplySupported` replace properties specified in `ApplySupportedDefaults`, collections are replaced as a whole

```

Comment:
```suggestion
- Primitive or collection-valued properties specified in `ApplySupported` replace the corresponding properties specified in `ApplySupportedDefaults`
- Complex-valued properties specified in `ApplySupported` override the corresponding properties specified in `ApplySupportedDefaults` using PATCH semantics recursively
```

Reply:
Accepted proposal even though we don't have complex properties yet in ApplySupportedDefaults - but we have them in Capabilities where we plan a similar change.